src/eko/couplings.py

```diff
@@ -195,7 +195,7 @@ def couplings_expanded(order, couplings_ref, nf, scale_from, scale_to):
 
 class Couplings:
     r"""
-        Computes the strong and electromagnetic coupling constants :math:`a_s,a`.
+        Computes the strong and electromagnetic coupling constants :math:`a_s, a_em`.
```

Comment:
Check the docstring also in practice! I'm quite sure `a_em` will be interpreted as `{a_e}m` which is not what you want ...